bin/admin/install

```diff
@@ -1127,6 +1127,47 @@ if [ "$nothing" = 0 ]; then
         action_done
     fi
 
+    # The Bastion > v3.04.00
+    action_doing "Migrating account creation metadata to new format where applicable"
+    at_least_one_changed=0
+    at_least_one_error=0
+    for accounthome in $(getent passwd | grep ":$basedir/bin/shell/osh.pl$" | cut -d: -f6); do
+        # if the old AND the new files exists, don't do anything: we won't delete the old file
+        # because we want our users to be able to roll back to an old version, and as this file doesn't
+        # hurt anything (it'll just get ignored by new versions), no point in inserting an incompatible change here.
+        if [ -f "$accounthome/accountCreate.comment" ] && ! [ -f "$accounthome/config.creation_info" ]; then
+            at_least_one_changed=$((at_least_one_changed + 1))
+            # build a config.creation_info file from the data stored in the legacy accountCreate.comment
+            perl -MJSON -MPOSIX -e '
+                my %h;
+                open(my $oldfh, "<", $ARGV[0]) or die $!;
+                while (<$oldfh>) {
+                    $h{$1}=$2 if m{^([^=]+)=(.+)};
```

Comment:
Depending on if the `m` in really useful here, this might be a `split('=', $_, 2)`

Reply:
done